src/iuse.cpp

```diff
@@ -3590,7 +3591,7 @@ int iuse::mp3( player *p, item *it, bool, const tripoint & )
     return it->type->charges_to_use();
 }
 
-const std::string &get_music_description()
+const std::string get_music_description()
```

Comment:
Please don't return a const value. See https://stackoverflow.com/questions/6299967 for why it's pointless. If you don't want to return a reference anymore, return a non-const value.

Reply:
Good point on not returning a const value, thanks.